content/en/account_management/billing/google_cloud.md

```diff
@@ -5,15 +5,15 @@ kind: documentation
 
 ## Overview
 
-Datadog bills for hosts running the Agent and all GCE instances picked up by the Google Cloud integration. You are not billed twice if you are running the Agent on a GCE instance picked up by the Google Cloud integration.
+Datadog bills for hosts running the Agent and all GCE instances picked up by the Google Cloud integration. Services outside of GCE, such as [Dataflow][6], may create billable GCE hosts in Datadog. To see which hosts are billed, navigate to the GCE page in the Google Cloud console. You are not billed twice if you are running the Agent on a GCE instance picked up by the Google Cloud integration.
 
 Other Google Cloud resources (CloudSQL, Google App Engine, Pub/Sub, etc.) are not part of monthly billing.
 
 ## Google Cloud metric exclusion
 
 Use the [Google Cloud integration tile][1] to control your metric collection. Go to the **Configuration** tab and select a project or add a new one. Each project is controlled under **Optionally Limit Metrics Collection to hosts with tag**. Limit metrics by [host tag][2]:
 
-{{< img src="account_management/billing/google-cloud01.png" alt="Google Cloud" >}}
+{{< img src="account_management/billing/google_cloud_metric_filter.png" alt="The Google Cloud page in Datadog, on the General tab, with the option to limit metric collection highlighted" >}}
```

Comment:
Good callout. Adding @smuhit to review this too in case there's some nuance around how this metrics collection filter works. (Furthermore, do we want to provide any guidance around filtering on specific parameters?)